tap_clickup/schemas/task.json

```diff
@@ -200,7 +200,7 @@
                             "type": ["string", "boolean", "null"]
                         },
                         "value": {
-                            "type": ["integer", "null"]
+                            "type": ["integer", "string", "boolean", "null"]
```

Comment:
snuck this in here as I was getting failures during testing due to data that had custom values with booleans / strings